src/graph_view.rs

```diff
@@ -229,13 +237,44 @@ impl<'a, N: Clone, E: Clone, Ty: EdgeType> GraphView<'a, N, E, Ty> {
             return;
         }
 
-        if !self.settings_interaction.selection_multi_enabled {
+        if !self.settings_interaction.node_selection_multi_enabled {
             self.deselect_all(comp);
         }
 
         self.select_node(idx);
     }
 
+    fn handle_edge_click(&mut self, idx: EdgeIndex, comp: &ComputedState) {
+        //TODO how should the settings be handled?
```

Comment:
I think the general behavioud should be the same as with nodes selection. 
* we click edge when edge clicking is enabled and select when clicking and selection is enabled (selection automatically turns clicking on)
* we deselect edge when clicking and selection is enabled and (edge is clicked or empty space is clicked)
* multiselect just allows to select multiple nodes/edges